src/main/resources/uischema/SwiftISO20022.json

```diff
@@ -0,0 +1,7 @@
+{
+  "connectorName": "swiftISO20022",
+  "connectionName": "SwiftISO20022",
+  "title": "SwiftISO20022 Connection",
+  "help": "<h1>WSO2 SWIFT ISO 20022 Mediation library</h1> <b>This library will provide functions to validate and convert Swift and ISO 20022 Message formats</b>",
+  "elements": []
+}
```

Comment:
Add EoF line

Reply:
Fixed in https://github.com/wso2-extensions/mediation-swift-iso20022-module/pull/2/commits/18ec29ed3926964e6b999c6fd5d385b24c1f8fa5